src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -433,6 +433,18 @@ public static class Parser
         private static readonly Regex AnimeReleaseGroupRegex = new Regex(@"^(?:\[(?<subgroup>(?!\s).+?(?<!\s))\](?:_|-|\s|\.)?)",
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
+        //Handle Exception Release Groups that don't follow -RlsGrp; Manual List
+        //groups whose releases end with RlsGroup) or RlsGroup]
+        /* Current list from Radarr 2021-08-11 PR#6494
+           Commenting out until seen in the wild; but implementing with ExceptionReleaseGroupRegexExact
+           Code in place to use this regex is commented out as well `exceptionMatch`
+        */
+        // private static readonly Regex ExceptionReleaseGroupRegex = new Regex(@"(?<releasegroup>(Tigole|Joy|YIFY|YTS.MX|YTS.LT|FreetheFish|afm72|Anna|Bandi|Ghost|Kappa|MONOLITH|Qman|RZeroX|SAMPA|Silence|theincognito|t3nzin|Vyndros|HDO)(?=\]|\)))", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+
+        //Handle Exception Release Groups that don't follow -RlsGrp; Manual List
+        // name only...BE VERY CAREFUL WITH THIS, HIGH CHANCE OF FALSE POSITIVES
+        private static readonly Regex ExceptionReleaseGroupRegexExact = new Regex(@"(?<releasegroup>D\-Z0N3|Fight-BB)", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
I wanna hear from @markus101 if he really wants to accept specific release groups hardcoded. There is some precedent for that.

Also D-Z0N3 is a movie release group, so doesn't belong here. And Fight-BB only use things like UFC.

Reply:
D-Z0N3 does in fact of at least one TV release.

`Life.2009.S01.1080p.BluRay.DTS5.1.x264-D-Z0N3`

Reply:
note for me - add an anchor to minimize possibility of screw ups.

/b seems to be the least impactful as these are for groups that do not follow sane standards